assets/src/scripts/components/hydrograph/index.js

```diff
@@ -129,32 +142,37 @@ const plotSvgDefs = function(elem) {
 };
 
 
-const plotLegend = function(elem, {displayItems, layout}) {
-    elem.select('.legend').remove();
-    let plotMarkers = createLegendMarkers(displayItems);
-    drawSimpleLegend(elem, plotMarkers, layout);
+const timeSeriesLegend = function(elem) {
+    elem.append('div')
+        .attr('class', 'hydrograph-container')
+        .append('svg')
```

Comment:
Why change to drawing the legend in a separate SVG?

Reply:
We will need to add at least some of the accessibility features to it such as a title and description.

Reply:
Yes, missed that.